pkgs/os-specific/linux/mwprocapture/default.nix

```
@@ -14,12 +11,12 @@ let
 
 in
 stdenv.mkDerivation rec {
-  name = "mwprocapture-1.2.${version}-${kernel.version}";
-  version = "4177";
+  name = "mwprocapture-1.3.0.${version}-${kernel.version}";
+  version = "4236";
 
   src = fetchurl {
     url = "http://www.magewell.com/files/drivers/ProCaptureForLinux_${version}.tar.gz";
-    sha256 = "1nf51w9yixpvr767k49sfdb9n9rv5qc72f5yki1mkghbmabw7vys";
+    sha256 = "1mfgj84km276sq5i8dny1vqp2ycqpvgplrmpbqwnk230d0w3qs74";
   };
 
   nativeBuildInputs = [ kernel.moduleBuildDependencies ];
```

not related to changes
```suggestion
  nativeBuildInputs = kernel.moduleBuildDependencies;
```
it's already a list
```
nix-repl> :l
Added 14203 variables.

nix-repl> linuxPackages.kernel.moduleBuildDependencies
[ «derivation /nix/store/nn99rfl37w94yn8ripdmlh4ar3iixvmn-libelf-0.8.13.drv» ]
```

---

Thanks!